app/components/elements/Checkbox.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+
```

Comment:
In general I agree, but I had significant trouble getting down to be able to manipulate the rebass component -- I'll take a look at the example that you pushed and see if I missed something when looking at it before

Reply:
I suppose it depends on what type of manipulation you need.  I'm imagining just changing the sizing/color/label fonts.  If there's more to it, it could get tricky, but then, I'd push back on any design requirements that force us to get tricky.